test/node.js

```diff
@@ -13,35 +13,28 @@ const expect = chai.expect
 const IPFSRepo = require('../src')
 
 describe('IPFS Repo Tests on on Node.js', () => {
-  const repos = [{
-    name: 'default',
-    opts: undefined,
-    init: false
-  }, {
-    name: 'memory',
-    opts: {
-      fs: require('interface-datastore').MemoryDatastore,
-      level: require('memdown'),
-      lock: 'memory'
+  const repos = [
+    {
+      name: 'default',
+      opts: undefined
     },
-    init: true
-  }]
+    {
+      name: 'memory',
+      opts: {
+        blockStore: require('interface-datastore').MemoryDatastore,
+        dataStore: require('interface-datastore').MemoryDatastore
+      }
+    }
+  ]
   repos.forEach((r) => describe(r.name, () => {
     const testRepoPath = path.join(__dirname, 'test-repo')
     const date = Date.now().toString()
     const repoPath = testRepoPath + '-for-' + date
-
     const repo = new IPFSRepo(repoPath, r.opts)
-
     before((done) => {
       series([
-        (cb) => {
-          if (r.init) {
```

Comment:
For a bad reason. Basically I thought it was an inconsistency in the way the stores were getting created and a bug in this test but now I see that the tests were failing because I dropped the `extension: ''` from the store options which was making the fs based tests fail because it couldn't find the files copied in the before step. Adding the extension back and reverting basically this whole file resolves it.